tests/integration/sniffer/capturer.py

```
@@ -0,0 +1,858 @@
+        udp_layer = packet[UDP]
+        payload = (
+            bytes(udp_layer.payload)
+            if hasattr(udp_layer, "payload") and udp_layer.payload
+            else b""
+        )
+        format_and_log_payload(
+            "UDP", src_ip, dst_ip, udp_layer.sport, udp_layer.dport, payload
+        )
+        result = analyze_udp_packet(packet)
+    elif packet.haslayer(TCP):
+        result = analyze_tcp_packet(packet)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)